docs/getting_started/basics/nesting_workflows.md

```diff
@@ -5,16 +5,21 @@ In Shift, any workflow can be embedded into a node and executed within another w
 ## Setting Workflow I/Os
 
 To create a reusable workflow, the input and output of the workflow must be specified. This is done through `Input` and `Output` nodes. The plugs created in these nodes must be `through` plugs and define which inputs are expected by the workflow and which are the outputs that will be returned.
-
-![Workflow I/Os](../../images/workflow_ios.png)
+<figure>
+      <img src="images/workflow_ios.png" alt="Workflow I/Os">
+      <figcaption><b>Figure 1</b>: Input and output nodes of a workflow.</figcaption>
+</figure>
 
 The plugs from these nodes will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa.
 
 ## Workflow Node
 
 The `Workflow` node allows to embed the execution of a workflow inside another workflow. This node has a *file* plug, where the workflow to nest is to be indicated. When the file plug content is set to reference a valid workflow file (with a *.sft* extension), the node will automatically update by creating the input and output plugs corresponding to the sourced workflow inputs and outputs. Then, when the `Workflow` node gets executed, it will execute the referenced workflow.
 
-![Nested Workflows](../../images/workflow_nesting.gif)
+<figure>
+      <img src="images/workflow_nesting.gif" alt="Workflow nesting">
```

Comment:
In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

Reply:
Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515